src/github/api/operations.rs

```diff
@@ -42,19 +41,7 @@ pub async fn merge_branches(
     commit_message: &str,
 ) -> Result<CommitSha, MergeError> {
     let client = repo.client();
-    let merge_url = repo
-        .repository
-        .merges_url
-        .as_ref()
-        .map(|url| url.to_string())
-        .unwrap_or_else(|| {
-            format!(
-                "{}/repos/{}/{}/merges",
-                base_github_url(),
-                repo.name().owner,
-                repo.name().name
-            )
-        });
+    let merge_url = format!("/repos/{}/{}/merges", repo.name().owner, repo.name().name);
```

Comment:
This (the fact that `octocrab` stopped supporting absolute URLs) is exactly the kind of error that can only be found with an integration test.

Reply:
Exactly.
Sorry, meant to split this into another PR for easier review, but I accidentally pushed it here